### DISCUSSION TITLE
Cards structure unification

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#630] [Proposals]
Actualmente la `data card` y la `media card` tienen estructuras de elementos un poco diferentes.

![image](https://user-images.githubusercontent.com/6722153/181269918-67a838b3-79fa-46e8-9e30-a1c27e99801c.png)

La data card tiene:
- Asset (icono)
- Tag
- Title
- _Subtitle_
- Description

La media card tiene: 
- Asset (imagen)
- Tag
- _Pretitle_
- Title
- Description

La diferencia entre ambas es que una tiene subtitle y la otra tiene pretitle, no tendría sentido que ambas cards tuviesen los mismos elementos?

**Comment by yceballost** [comment]
Nombro a @gitbraga porque creo que es algo que en Brasil han necesitado

**Comment by AnaMontes11** [comment]
¿Cual fue la lógica para que cada card tuviera esa definición tan concreta? sigue aplicando a día de hoy? yo sí daría la posibilidad de que ambas cards tuvieran esos elementos.

**Reply by yceballost** [reply]
Si no recuerdo mal, @HectorSancho, la idea era que ambas cards tuviesen diferencias en cuanto al uso que se les daba. Yo creo que eso lo marca el asset y no tanto la estructura de estos elementos.

**Reply by HectorSancho** [reply]
La definición se basó en una recopilación bastante grande de casos de uso en Novum y otros productos. Pero como hemos comentado más de una vez se podrían unificar.
